documentation/docs/user-guide/user-officer/proposal.md

```diff
@@ -0,0 +1,16 @@
+# Proposal
+
+
+## Edit a proposal 
+
+# How do I edit a proposal?
+
+**1. Find the proposal you wish to edit**
```

Comment:
The generated list is not shown correctly when there is no blank line at the beginning in some cases. Could you add a new line wherever it's needed?

<img width="809" alt="Screenshot 2024-07-25 at 10 14 00" src="https://github.com/user-attachments/assets/6a057afb-cdbc-4ce3-bf3c-c08027bc589a">
<img width="647" alt="Screenshot 2024-07-25 at 10 14 09" src="https://github.com/user-attachments/assets/58c6693f-03af-4127-b795-8748018df52f">